content/en/seachat/seachat-manual/02-create-agent/04-agent-information.md

```diff
@@ -0,0 +1,114 @@
+---
+title: "Agent Information"
+description: ""
+lead: ""
+date: 2024-04-26T08:48:45+00:00
+lastmod: 2024-05-19T08:48:45+00:00
+weight: 90
+draft: false
+images: []
+toc: true
+aliases:
+  - /en/seachat/seachaat-manual/02-create-agent/03-agent-memory
+---
+
+# Overview
+
+Agent information is the core of your AI agent. It contains all the necessary information about your agent, including its name, description, and other advanced settings like memory fields. In this tutorial, we will walk you through each of the parameters that you will find in **Agent Information** under **Agent Configuration**.
+
+---
+
+## Basic Setting in Agent Information
+
+SeaChat allows you to configure your AI agent's information such as names, use case, and description. Inside **Basic Setting**, users can set the following parameters:
+
+### Name
+The name of your AI agent. This name will be displayed in the conversation window.
+
+> :page_facing_up: **NOTE**
+> 
+> This will be the name of your agent for the purpose og agent management. If you wish to change the displayed name in your agent integration. Please refer to [Webpage Integration](/seachat/seachat-manual/04-channels/08-install-to-webpage/).
+
+
+### Use Case
+The use case of your AI agent. This will help you to categorize your AI agent. SeaChat provides a list of use cases to choose from. Depending on your needs, you can choose a use case that best fits your AI agent. Different use cases might require different description settings, but you can simply follow the instructions in the description section when incorporating a use case.
+
+<div style="display: flex; flex-direction: column; align-items: center;">
+<div style="width: 100%; text-align: center; display: flex; flex-direction: column; align-items: center; justify-item: center">
+    <a href="/images/seachat/en/agent-information/use-case-examples.png" target="_blank">
+    <img width="80%" style="border-radius: 0.4rem; cursor: zoom-in;" src="/images/seachat/en/agent-information/use-case-examples.png" alt="image that display the use case options available in SeaChat">
+    </a>
+</div>
+    <p style="margin-top: 20px; font-size: 15px">Choose a use case for your AI agent</p>
+</div>
+
+### Description
```

Comment:
Add one more detail: the use case comes with a basic prompt. Description is where you can customize the bot even more by appending/inserting a description about your agent into the prompt. To view full prompt, you can check the right-hand side of the page